pkg/reloader/example_test.go

```diff
@@ -0,0 +1,54 @@
+package reloader_test
```

Comment:
I would put it in `reloader_example_test` package?

Reply:
I think the name of package should be same as official.  Just `<package>_test`.
For example: https://golang.google.cn/src/container/heap/example_intheap_test.go